Managers/PerformanceMan.cpp

```diff
@@ -139,7 +139,7 @@ namespace RTE {
 			g_FrameMan.GetLargeFont()->DrawAligned(&bitmapToDrawTo, c_StatsOffsetX, c_StatsHeight, str, GUIFont::Left);
 
 			// Display the average
-			std::snprintf(str, sizeof(str), "MSPF: %zi", m_MSPFAverage);
+			std::snprintf(str, sizeof(str), "MSPF: %i", m_MSPFAverage);
```

Comment:
%zi was a typo right?
I don't remember all the specifiers but I'm not seeing anything suggesting that's allowed when I do a quick google.

Reply:
Must've been. Was causing warnings, so I fixed it.